get_linux/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<head>
+<title>Get QField for Windows</title>
+</head>
+<body>
+<script type="text/javascript">
+fetch('https://api.github.com/repos/opengisch/QField/releases/latest')
+.then(response => {
+    if (response.status !== 200) throw new Error();
+    return response.json();
+})
+.then(json => { window.location.replace('https://github.com/opengisch/QField/releases/download/'+json['tag_name']+'/qfield-'+json['tag_name']+'-linux-x64.AppImage') })
+.catch(error => { window.location.replace('https://github.com/opengisch/QField/releases/latest') });
+</script>
+<p>Redirecting to the windows download page. If this does not work please go to the <a href='https://github.com/opengisch/QField/releases/latest'>latest release page</a>.</p>
```

Comment:
```suggestion
<p>Redirecting to the ${queryParams.nickname} download page. If this does not work please go to the <a href='https://github.com/opengisch/QField/releases/latest'>latest release page</a>.</p>
```
We can pass the platform name also as a queryParameter. Needs a bit more js to work.

Reply:
Actually there I'd just no refer to the platform name.